### PR TITLE
Support for command line triggers

### LINKED ISSUE #87: Support for Command Line Triggers
Resolves https://github.com/checkly/public-roadmap/issues/158

### PR DESCRIPTION
**Pull request by pilimartinez**
Resolves #87

### CODE REVIEW

**Comment by umutuzgur** [comment]
@ianaya89 I'm leaving this one to you. I'm a bit busy to check this PR at the moment

File: checkly/resource_trigger_group.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+		if strings.Contains(err.Error(), "404") {
+			d.SetId("")
+			return nil
+		}
+		return fmt.Errorf("resourceTriggerGroupRead: API error: %w", err)
+	}
+	return resourceDataFromTriggerGroup(mw, d)
+}
+
+func resourceTriggerGroupUpdate(d *schema.ResourceData, client interface{}) error {
```
Comment (ianaya89) [inline checkly/resource_trigger_group.go:104]: I think we should not do anything in the update methods, no API and just return the resource.

**Review by ianaya89 — CHANGES_REQUESTED** [review]

File: checkly/resource_trigger_group.go (hunk 1 above)
Comment (pilimartinez) [reply checkly/resource_trigger_group.go:104]: Okay, but how can I return the resource without calling the API?

Comment (ianaya89) [reply checkly/resource_trigger_group.go:104]: We already have the data, you just need to create the resource type and set the values

**Review by ianaya89 — APPROVED** [review]

**Pull request merged**